Source/WebCore/css/parser/CSSParserContext.h

```diff
@@ -62,42 +62,42 @@ struct CSSParserContext {
     String charset;
     CSSParserMode mode { HTMLStandardMode };
     std::optional<StyleRuleType> enclosingRuleType;
-    bool isHTMLDocument { false };
+    bool isHTMLDocument : 1 { false };
```

Comment:
This is a temporary stack object, does the size matter that much?

Reply:
Yeah, but it is showing up as a sample sometimes. Anyway, I think it is good not to make it super large.